src/playback/internal/playbackuiactions.cpp

```diff
@@ -108,6 +125,34 @@ const UiActionList PlaybackUiActions::m_mainActions = {
              TranslatableString("action", "Set playback level"),
              IconCode::Code::AUDIO
              ),
+    UiAction("action://playback/change-api",
```

Comment:
A variable (constant) is declared there, but it is not used here, maybe then remove it.